src/index.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const state = {
+  tempValue: 65,
+};
+
+const updateTempValue = () => {
+  const tempValue = document.getElementById('tempValue');
+  const img = document.getElementById('landscape');
+  const temp = state.tempValue;
+
+  tempValue.textContent = temp;
+  tempValue.className = '';
+
+  if (temp < 32) {
+    tempValue.classList.add('temp-cold');
+    img.src = 'assets/below32.jpg';
+  } else if (temp < 45) {
+    tempValue.classList.add('temp-cool');
+    img.src = 'assets/32-45.jpg';
+  } else if (temp < 72) {
+    tempValue.classList.add('temp-mild');
+    img.src = 'assets/45-72.jpg';
+  } else if (temp < 85) {
+    tempValue.classList.add('temp-warm');
+    img.src = 'assets/72-85.jpg';
+  } else if (temp <= 100) {
+    tempValue.classList.add('temp-hot');
+    img.src = 'assets/85-100.jpg';
+  } else {
+    tempValue.classList.add('temp-extreme');
+    img.src = 'assets/above100.jpg';
+  }
```

Comment:
When I read the name `updateTempValue` I think that a temperature is going to get changed, but when I read the implementation, I see that not only is temperature set, but the display is also updated.

I'd prefer the logic on lines 12-33 be in a helper function, like `updateDisplay`

```suggestion
const updateTempValue = () => {
  const tempValue = document.getElementById('tempValue');
  const img = document.getElementById('landscape');
  const temp = state.tempValue;

  updateDisplay();
```

Reply:
It was implemented in the new_version branch.